internal/services/monitor/monitor_activity_log_alert_resource.go

```diff
@@ -103,31 +105,112 @@ func resourceMonitorActivityLogAlert() *pluginsdk.Resource {
 								"Error",
 								"Critical",
 							}, false),
+							ConflictsWith: []string{"criteria.0.levels"},
+						},
+						"levels": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							Elem: &pluginsdk.Schema{
+								Type: pluginsdk.TypeString,
+								ValidateFunc: validation.StringInSlice([]string{
+									"Verbose",
+									"Informational",
+									"Warning",
+									"Error",
+									"Critical",
```

Comment:
are there not constants for these in the SDK? if not could we open an issue on the rest specs to have them added?

Reply:
The API only specifies the condition should be [key-value pairs](https://learn.microsoft.com/en-us/rest/api/monitor/activity-log-alerts/create-or-update?tabs=HTTP#alertruleanyoforleafcondition), and there is no enum for values. 

And more details on property mappings:
`level` corresponds to:
```json
{
      "field": "level",
      "equals": "Verbose"
},
```
`levels` corresponds to:
```json
{
    "anyOf": [
        {
            "field": "level",
            "equals": "Verbose"
        },
        {
            "field": "level",
            "equals": "Informational"
        }
    ]
},
```

or 

```json
{
      "field": "level",
      "containsAny": ["Verbose", "Informational"]
},
```

since `levels` may map to 2 REST API schemas, here I am using the `anyOf` one in create/update, but accept either one in read.